redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1378 @@
+                return True
+
+    @commands.command()
+    @commands.guild_only()
+    @checks.mod_or_permissions(manage_roles=True)
+    async def activemutes(self, ctx: commands.Context):
+        """
+        Displays active mutes on this server.
+        """
+
+        msg = ""
+        if ctx.guild.id in self._server_mutes:
+            mutes_data = self._server_mutes[ctx.guild.id]
+            if mutes_data:
+
+                msg += _("__Server Mutes__\n")
+                for user_id, mutes in mutes_data.items():
+                    if not mutes:
+                        continue
+                    user = ctx.guild.get_member(user_id)
+                    if not user:
+                        user_str = f"<@!{user_id}>"
+                    else:
+                        user_str = user.mention
+                    if mutes["until"]:
+                        time_left = timedelta(
+                            seconds=mutes["until"] - datetime.now(timezone.utc).timestamp()
+                        )
+                        time_str = humanize_timedelta(timedelta=time_left)
+                    else:
+                        time_str = ""
+                    msg += f"{user_str} "
+                    if time_str:
+                        msg += _("__Remaining__: {time_left}\n").format(time_left=time_str)
+                    else:
+                        msg += "\n"
+        for channel_id, mutes_data in self._channel_mutes.items():
+            if not mutes_data:
+                continue
+            if ctx.guild.get_channel(channel_id):
+                msg += _("__<#{channel_id}> Mutes__\n").format(channel_id=channel_id)
+                for user_id, mutes in mutes_data.items():
+                    if not mutes:
+                        continue
+                    user = ctx.guild.get_member(user_id)
+                    if not user:
+                        user_str = f"<@!{user_id}>"
+                    else:
+                        user_str = user.mention
+                    if mutes["until"]:
+                        time_left = timedelta(
+                            seconds=mutes["until"] - datetime.now(timezone.utc).timestamp()
+                        )
+                        time_str = humanize_timedelta(timedelta=time_left)
+                    else:
+                        time_str = ""
+                    msg += f"{user_str} "
+                    if time_str:
+                        msg += _("__Remaining__: {time_left}\n").format(time_left=time_str)
+                    else:
+                        msg += "\n"
+        if msg:
+            for page in pagify(msg):
+                await ctx.maybe_send_embed(page)
+            return
+        await ctx.maybe_send_embed(_("There are no mutes on this server right now."))
+
+    @commands.command(usage="<users...> [time_and_reason]")
+    @commands.guild_only()
+    @checks.mod_or_permissions(manage_roles=True)
+    async def mute(
```

Comment:
To add here, the issue with failure reasons can also be seen with role-based mutes, although role-based mutes tend to have less permission-related issues so it's not as hard to read, but still, in case of different reason it can be non-obvious what reason refers to what:
![image](https://user-images.githubusercontent.com/6032823/97060879-216c7b00-1595-11eb-9eb6-9be9e4feac9f.png)

Grouping reasons still seems like the best idea for addressing it to me, but feel free to figure out something else that makes it more user-friendly.